notebooks/20220328_correlations_mrnorm.ipynb

```diff
@@ -0,0 +1,518 @@
+      "text/plain": [
+       "plot without title"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "height": 420,
+       "width": 900
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mrnorm_hlog_pa14_kmers_cor_df <- cor_to_long(mrnorm_hlog_pa14_kmers_cor) %>%\n",
+    "  left_join(pa14_hash_map_operon_small, by = c(\"row\" = \"minhash\")) %>% # determine which operon, if any, the row is part of\n",
+    "  select(row, col, corr, operon_id_row = operon_id) %>% # record operon identity of row\n",
+    "  left_join(pa14_hash_map_operon_small, by = c(\"col\" = \"minhash\")) %>% # determine which operon the col is part of\n",
+    "  select(row, col, corr, operon_id_row, operon_id_col = operon_id) %>% # record operon identity of col\n",
+    "  mutate(operon = ifelse(operon_id_row == operon_id_col, \"operon pair\", \"not operon pair\")) %>% # label operon pairs\n",
+    "  mutate(operon = ifelse(is.na(operon), \"not operon pair\", operon))\n",
+    "\n",
+    "ggplot(mrnorm_hlog_pa14_kmers_cor_df, aes(x = corr, fill = operon)) +\n",
+    "  geom_histogram(alpha = .5) +\n",
+    "  theme_classic()"
```

Comment:
So I think this is a little bit expected. We saw with the Pseudomonas compendium stuff that you have to log the data after normalization to account for high abundance outliers, which will drive trends otherwise. 

The first peak is centered-ish around zero. This peak represents all of the k-mers whose abundances are not correlated. This peak would be much taller if I hadn't filtered the k-mers so stringently -- so if I had included more k-mers, there would be a larger number of correlations that had low values. 
 
The second peak are the things that are actually correlated. Unfortunately, while some of these k-mers are correlated because they are part of the same operon, and therefore expressed together, there's a bunch of things that are correlated that are not part of operons.

I think I need to check if regulons make up part of this...